lib/clearance/password_strategies/bcrypt.rb

```diff
@@ -19,18 +22,20 @@ def password=(new_password)
         @password = new_password
 
         if new_password.present?
-          cost = if defined?(::Rails) && ::Rails.env.test?
-                   ::BCrypt::Engine::MIN_COST
-                 else
-                   ::BCrypt::Engine::DEFAULT_COST
-                 end
-
           self.encrypted_password = ::BCrypt::Password.create(
             new_password,
             cost: cost,
           )
         end
       end
+
+      def cost
+        if defined?(::Rails) && ::Rails.env.test?
+          ::BCrypt::Engine::MIN_COST
+        end
+        # Otherwise return nil, so that the cost stored in BCrypt
```

Comment:
What do you think about being explicit here and adding an else branch that return `BCrypt::Engine.cost`?